AdyenNetworking/Helpers/URLSessionHelpers.swift

```diff
@@ -14,35 +14,21 @@ internal struct URLSessionSuccess {
 
 /// :nodoc:
 internal extension URLSession {
-    /// :nodoc:
-    func dataTask(with url: URL, completion: @escaping ((Result<URLSessionSuccess, Error>) -> Void)) -> URLSessionDataTask {
-        dataTask(with: url, completionHandler: { data, response, error in
-            self.handle(data: data, response: response, error: error, completion: completion)
-        })
-    }
-
-    /// :nodoc:
-    func dataTask(with urlRequest: URLRequest, completion: @escaping ((Result<URLSessionSuccess, Error>) -> Void)) -> URLSessionDataTask {
-        dataTask(with: urlRequest, completionHandler: { data, response, error in
-            self.handle(data: data, response: response, error: error, completion: completion)
-        })
-    }
 
     /// :nodoc:
-    private func handle(data: Data?, response: URLResponse?, error: Error?, completion: @escaping ((Result<URLSessionSuccess, Error>) -> Void)) {
-        let httpResponse = response as? HTTPURLResponse
-        if let headers = httpResponse?.allHeaderFields,
-           let path = response?.url?.path {
-            adyenPrint("---- Response Headers (/\(path)) ----")
-            adyenPrint(headers)
-        }
```

Comment:
we should log the raw data response, its handy for debugging

Reply:
sorry I meant the headers, we need to print the headers, for example to see the pspReference and other information.

Reply:
we print them in line 164 i believe